_data/sanidhyamangal/sanidhyamangal.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sanidhya Mangal # No longer than 18 characters
+institution: Medi-Caps University # no longer than 58 characters
+profile_pic: sanidhyamangal.jpg # Relative path to the image you uploaded
+quote: "Future is more predictable with data and computers" # No longer than 100 characters
```

Comment:
I just noticed that you have double quotes(`"`) here. Would you mind removing that to guarantee the format remains the same for all the PRs?

Reply:
Removed.